recipes/hande/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -ex
+
+
+if [[ "$mpi" == "nompi" ]]; then
+
+cmake ${CMAKE_ARGS} -H. -Bbuild \
+    -DCMAKE_INSTALL_PREFIX=${PREFIX} \
```

Comment:
This should do the trick (general CMake feature)
```suggestion
    -DCMAKE_INSTALL_PREFIX=${PREFIX} \
    -DBUILD_SHARED_LIBS=ON
```

Reply:
Cheers!